.gitignore

```diff
@@ -21,3 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.history
```

Comment:
Git should ignore all files & folders inside .history 
change .history to .history/